recipes/r-duckplyr/meta.yaml

```
@@ -0,0 +1,100 @@
+{% set version = '0.4.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-duckplyr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/duckplyr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/duckplyr/duckplyr_{{ version }}.tar.gz
+  sha256: 3e23255a91f9ebc8a8e0fb2fb32d96e16fa7620527d912cd8ec13fa3f32d2fd3
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-dbi
+    - r-cli
+    - r-collections
+    - r-dplyr >=1.1.4
+    - r-duckdb >=0.10.2
+    - r-glue
+    - r-jsonlite
+    - r-lifecycle
+    - r-rlang >=1.0.6
+    - r-tibble
+    - r-tidyselect
+    - r-vctrs >=0.6.3
+  run:
+    - r-base
+    - r-dbi
+    - r-cli
+    - r-collections
+    - r-dplyr >=1.1.4
+    - r-duckdb >=0.10.2
+    - r-glue
+    - r-jsonlite
+    - r-lifecycle
+    - r-rlang >=1.0.6
+    - r-tibble
+    - r-tidyselect
+    - r-vctrs >=0.6.3
+
+test:
+  commands:
+    - $R -e "library('duckplyr')"           # [not win]
+    - "\"%R%\" -e \"library('duckplyr')\""  # [win]
+
+about:
+  home: https://duckdblabs.github.io/duckplyr/, https://github.com/duckdblabs/duckplyr
```

Let's split the URLs by their purpose (makes for cleaner README and future autotick PRs):

```suggestion
  home: https://duckdblabs.github.io/duckplyr/
  dev_url: https://github.com/duckdblabs/duckplyr
```